rosidl_generator_c/test/test_interfaces.c

```diff
@@ -333,8 +337,39 @@ int test_strings(void)
   return 0;
 }
 
-
 /**
+ * Test message with different wstring types.
+ */
+int test_wstrings(void)
+{
+  bool res = false;
+  rosidl_generator_c__msg__WStrings * wstrings = NULL;
+
+  wstrings = rosidl_generator_c__msg__WStrings__create();
+  EXPECT_NE(wstrings, NULL);
+
+  res = rosidl_generator_c__U16String__assign(&wstrings->empty_wstring, TEST_WSTRING);
+  EXPECT_EQ(true, res);
+  // EXPECT_EQ(0, strcmp(wstrings->empty_wstring.data, TEST_WSTRING));
```

Comment:
What are the issues with these tests? Maybe add a TODO explaining.

Reply:
Added in 45b923db0afc7f164473b84f9c61f4c0ad94dc6c.

That will also cover this comment: https://github.com/ros2/test_interface_files/pull/4#discussion_r280528815